packages/infra/config/env-config.ts

```diff
@@ -266,6 +266,18 @@ type EnvConfigBase = {
       };
     };
   };
+  surescripts?: {
+    surescriptsReplicaBucketName: string;
+    surescriptsBundleBucketName: string;
+    surescriptsSenderId: string;
+    surescriptsReceiverId: string;
+    surescriptsHost: string;
+    secrets: {
+      SURESCRIPTS_SFTP_SENDER_PASSWORD: string;
+      SURESCRIPTS_SFTP_PUBLIC_KEY: string;
```

Comment:
we have the public keys directly in configs instead of in secrets. would just keep following that pattern

Reply:
discussed irl, resolved